aplib.net-demo/Assets/Scripts/Camera/CameraManager.cs

```diff
@@ -18,5 +20,7 @@ private void Awake()
         else
             Instance = this;
         DontDestroyOnLoad(gameObject);
+
+        PlayerTransform = PlayerVisTransform.parent;
```

Comment:
Yeah, this can be kind of cursed in some instances, please refer to my comment above.